### PR TITLE
Freeze werkzeug to version<2.2.3

### LINKED ISSUE #3916: Quote and encoding error
Tests are failing, probably due to some update lib. One example:

FAILED tests/server/blueprints/clinvar/test_clinvar_views.py::test_clinvar_update_submission - TypeError: quote() doesn't support 'encoding' for bytes

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3916 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
- [x] All tests should pass

**Review:**
- [ ] code approved by
- [x] tests executed by Github actions

### CODE REVIEW

**Review by dnil — APPROVED** [review]
🥶 Agreed, but we should fix it eventually, and probably consider leaving py 3.7 to rest as well. I notice we have 3.11 on the general hasta stage, and I think keeping the 3.7 support mainly was for hasta and probably the old clinical-db?

**Comment by northwestwitch** [comment]
> 🥶 Agreed, but we should fix it eventually, and probably consider leaving py 3.7 to rest as well. I notice we have 3.11 on the general hasta stage, and I think keeping the 3.7 support mainly was for hasta and probably the old clinical-db?

yes, we should definitely fix it so we can use updated libs! Regarding hasta, yes, probably!

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Everything needed in production
-werkzeug
+werkzeug<2.2.3
```
Comment (dnil) [inline requirements.txt:2]: Did you try <=2.2.3 by the way? I know I have 2.2.2, but..

Comment (northwestwitch) [reply requirements.txt:2]: Yes I'm using it locally. We might not be using the specific problematic code in scout, but let's fix it!

**Pull request merged**